scripts/qa_crowbarsetup.sh

```diff
@@ -2410,6 +2410,13 @@ function onadmin_teardown()
 ruby=/usr/bin/ruby
 iscloudver 5plus && ruby=/usr/bin/ruby.ruby2.1
 
+tftpboot_repos_dir=/srv/tftpboot/repos
+tftpboot_suse_dir=/srv/tftpboot/suse-$suseversion
```

Comment:
Let's just hard-code for now. We do that for 12.0 already anyway, and I agree it's unlikely we will use anything else than 11.3.